internal/core/cobra_utils_test.go

```diff
@@ -53,3 +70,88 @@ func Test_handleUnmarshalErrors(t *testing.T) {
 		),
 	}))
 }
+
+func Test_PositionalArg(t *testing.T) {
+	t.Run("Error", func(t *testing.T) {
+		t.Run("Missing1", Test(&TestConfig{
+			Commands: testGetCommands(),
+			Cmd:      "scw test-positional",
+			Check: TestCheckCombine(
+				TestCheckExitCode(1),
+				TestCheckError(&CliError{
+					Err:  fmt.Errorf("a positional argument is required for this command"),
+					Hint: "Try running: scw test-positional <name-id>",
+				}),
+			),
+		}))
+
+		t.Run("Missing2", Test(&TestConfig{
+			Commands: testGetCommands(),
+			Cmd:      "scw test-positional tag=world",
+			Check: TestCheckCombine(
+				TestCheckExitCode(1),
+				TestCheckError(&CliError{
+					Err:  fmt.Errorf("a positional argument is required for this command"),
+					Hint: "Try running: scw test-positional <name-id> tag=world",
+				}),
+			),
+		}))
+
+		t.Run("Invalid1", Test(&TestConfig{
+			Commands: testGetCommands(),
+			Cmd:      "scw test-positional name-id=plop tag=world",
+			Check: TestCheckCombine(
+				TestCheckExitCode(1),
+				TestCheckError(&CliError{
+					Err:  fmt.Errorf("a positional argument is required for this command"),
```

Comment:
what do you think of a different error, something like : 

`'name-id' is a positional argument` ?